index.js

```diff
@@ -209,7 +208,9 @@ const lint = () => {
 	const linter = (atom.config.get('jshint.supportLintingJsx') || atom.config.get('jshint.transformJsx')) ? jsxhint().JSXHINT : jshint().JSHINT;
 
 	const origCode = editor.getText();
-	const code = editor.getGrammar().scopeName === 'source.jsx' ? reactDomPragma(origCode) : origCode;
+	const grammarScope = editor.getGrammar().scopeName;
+	const isJsx = grammarScope === 'source.jsx' || grammarScope === 'source.js.jsx';
```

Comment:
This is a weird modification: it's because after testing some files, this scope name came up several times.